### ISSUE TITLE
How to set defaults?

### ISSUE THREAD

**Issue by bryankaplan** [shivaprsd/doqment#5]
With pdfjs, I can go into `about:config` and set `pdfjs.defaultZoomValue` to `'page-width'`.

Is there an equivalent means to set the default zoom value in doqment?

More broadly, is there any means to set any default values?

**Comment by shivaprsd** [comment]
@bryankaplan No, and yes.

There is a way, but you have to use the browser Dev Tools. Open a _doqment_ tab, open the Console and execute the following code:

```js
PDFViewerApplication.preferences.set('defaultZoomValue', 'page-width')
```

Similarly for any other property. You will have to do this only once.

(Unfortunately this won't have any effect at the moment; I am overriding the defaults to force-enable the annotation editor in the current version of _doqment_. This is reverted in the upcoming release, which should be available in the store in a few days time.)

---

There is already a request in #2 for a convinient way to do this. I [proposed](https://github.com/shivaprsd/doqment/issues/2#issuecomment-1263337533) to implement a UI, but it got [deferred](https://github.com/shivaprsd/doqment/issues/2#issuecomment-1271101257). I hope to work on it in the near future. 🙂

**Comment by bryankaplan** [comment]
Thank you, @shivaprsd. I agree an Options page is the right answer, once you're able to build it out. In the meantime, that's an easy enough workaround. I look forward to being able to set the pref. Thank you, btw, for doqment — I love it.

**Comment by shivaprsd** [comment]
@bryankaplan The release is now published in Firefox Add-on store. Please update.

**Comment by bryankaplan** [comment]
Updated, tested, and confirmed — LGTM. Thank you, @shivaprsd.

**Comment by shivaprsd** [comment]
@bryankaplan We finally have an Options page! Please update to version 0.7.

Better late than never I guess. 🙂

**Comment by bryankaplan** [comment]
It looks great, @shivaprsd. It's so helpful. Thank you!
